algorithms/pact/pact_ops.py

```diff
@@ -3066,11 +3146,52 @@ def forward(self,x,y):
         #         return x
         #     return x * self.Delta
 
+        if not isinstance(y, torch.Tensor):
+            raise Exception("IntegerDiv trying to divide by const!")
+
         if self.integer_node:
             return self.MyIntegerDiv.apply(x,y,int(self.Delta.item()),int(self.eps.item()), int(self.eta.item()))
         else:
             return self.MyIntegerDiv.forward(None, x,y,self.Delta,self.eps, self.eta)
 
+class PACTTrueIntegerDiv(nn.Module):
+
+    class MyTrueIntegerDiv(torch.autograd.Function):
+
+        @staticmethod
+        def forward(ctx, x, y, Delta, eps, eta):
+            return torch.floor((x * Delta * eta / (y*eta + eps)) + 0.5)
+
+        @staticmethod
+        @parse_args('v','i','i', 'i', 'i')
+        def symbolic(g, x, y, Delta, eps, eta):
+            return g.op("PACTOps::TrueIntegerDiv", x, y_i = y, Delta_i = Delta, eps_i=eps, eta_i = eta)
+
+    def __init__(self, Delta, integer_node=True, eps=1., eta=1.):
+        super().__init__()
+        self.register_buffer('Delta',torch.Tensor((int(Delta),)))
+        self.register_buffer('eps',torch.Tensor((int(eps),)))
+        self.register_buffer('eta', torch.Tensor((int(eta),)))
+        self.integer_node = integer_node
+
+    def forward(self,x,y):
+        # SCHEREMO: Shortcut degenerate cases (y == 1, eps == 0)
```

Comment:
Remove this comment, please.

Reply:
Done in [this commit](https://github.com/pulp-platform/quantlib/pull/15/commits/23624bfbe97ad31c78ca27cf113bc74aed87e5ea)